<launch>

    <node name="bl_fcu" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 0 0 0 1 base_link fcu" />

    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_odom" clear_params="true">
        <param name="frequency" value="30"/>

        <param name="odom0" value="vision_estimate"/>
        <param name="odom1" value="apriltag_estimate"/>
        <param name="imu0" value="/mavros/imu/data"/>

        <rosparam param="odom0_config">[true, true, true,
                                        true, true, true,
                                        false, false, false,
                                        false, false, false,
                                        false, false, false]</rosparam>
        <rosparam param="odom1_config">[true, true, true,
                                        false, false, false,
                                        false, false, false,
                                        false, false, false,
                                        false, false, false]</rosparam>
        <rosparam param="imu0_config">[false, false, false,
                                        true, true, true,
                                        false, false, false,
                                        false, false, false,
                                        true, true, true,
                                        true, true, true]</rosparam>
    </node>
</launch>
